### PR TITLE
chore: remove deprecated lsif_indexable_repositories table

### PR DESCRIPTION
**Pull request by tjdevries**

### CODE REVIEW

**Comment by tjdevries** [comment]
Do I actually have to have the down migration re-create the table if it's not being referenced in this version at all? It just feels gross :'(

**Review by efritz — APPROVED** [review]

**Comment by efritz** [comment]
> Do I actually have to have the down migration re-create the table if it's not being referenced in this version at all? It just feels gross :'(

@tjdevries If we don't then we break a down migration path back to a version that does use it

File: migrations/frontend/1528395857_drop_lsif_indexable_repositories.down.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+BEGIN;
+
+CREATE TABLE lsif_indexable_repositories (
+    id integer NOT NULL,
```
Comment (efritz) [inline migrations/frontend/1528395857_drop_lsif_indexable_repositories.down.sql:5]:
```suggestion
    id SERIAL PRIMARY KEY NOT NULL,
```

File: migrations/frontend/1528395857_drop_lsif_indexable_repositories.down.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    repository_id integer NOT NULL,
+    search_count integer DEFAULT 0 NOT NULL,
+    precise_count integer DEFAULT 0 NOT NULL,
+    last_index_enqueued_at timestamp with time zone,
+    last_updated_at timestamp with time zone DEFAULT now() NOT NULL,
+    enabled boolean
+);
+
```
Comment (efritz) [inline migrations/frontend/1528395857_drop_lsif_indexable_repositories.down.sql:13]:
```suggestion

CREATE UNIQUE INDEX lsif_indexable_repositories_repository_id_key ON lsif_indexable_repositories (repository_id);

```

**Review by efritz — CHANGES_REQUESTED** [review]
Can you apply these changes, then do a migrate up 1/down 1/up 1 to make sure there's no errors?

**Comment by tjdevries** [comment]
K, ran up & down migrations and seemed fine.

**Review by efritz — APPROVED** [review]

**Pull request merged**